src/components/AdjMatrix/AdjMatrixMethods.ts

```diff
@@ -49,6 +49,7 @@ export class View {
   private highlightedNodes: { [key: string]: any[] } = {};
   private columnSelectedNodes: any[] = [];
   private mouseOverEvents: any;
+  private maxVal: number = 0
```

Comment:
We should probably use a more descriptive variable name for this. You'll also need to propagate this change through the rest of the code.
```suggestion
  private maxNumConnections: number = 0
```